tests/Kernel/CorporateBlocks/CorporateBlocksFooterTest.php

```diff
@@ -5,316 +5,182 @@
 
-    $first_column = $crawler->filter('footer.ecl-footer div.ecl-footer__sections div.ecl-row section.ecl-footer__section:nth-child(1)');
+    $actual = $section->filter('a.ecl-footer-core__title');
+    $this->assertEquals($test_data['site_name']['label'], $actual->text());
+    $this->assertEquals($test_data['site_name']['href'], $actual->attr('href'));
 
-    $actual = $first_column->filter('h1.ecl-footer__section-title');
-    $this->assertEquals($test_data['about_ec_title'], trim($actual->text()));
+    $actual = $section->filter('div.ecl-footer-core__description');
+    $expected = new FormattableMarkup('This site is managed by the @name', ['@name' => 'ACP–EU Joint Assembly']);
+    $this->assertEquals($expected, $actual->text());
 
-    $actual = $first_column->filter('ul li:nth-child(1) > a');
-    $this->assertEquals($test_data['about_ec_links'][0]['href'], $actual->attr('href'));
-    $this->assertEquals($test_data['about_ec_links'][0]['label'], $actual->text());
+    $section = $crawler->filter('footer.ecl-footer-core section.ecl-footer-core__section2');
+    $this->assertLinkList($section, 'core', $test_data['class_navigation']);
 
-    $actual = $first_column->filter('ul li:nth-child(2) > a');
-    $this->assertEquals($test_data['about_ec_links'][1]['href'], $actual->attr('href'));
-    $this->assertEquals($test_data['about_ec_links'][1]['label'], $actual->text());
+    $section = $crawler->filter('footer.ecl-footer-core section.ecl-footer-core__section3');
+    $this->assertLinkList($section, 'core', $test_data['service_navigation']);
 
-    $second_column = $crawler->filter('footer.ecl-footer div.ecl-footer__sections div.ecl-row section.ecl-footer__section:nth-child(2)');
-    $actual = $second_column->filter('h1.ecl-footer__section-title');
-    $this->assertEquals($test_data['social_media_title'], trim($actual->text()));
-
-    $actual = $second_column->filter('ul li:nth-child(1) > a');
-    $this->assertEquals($test_data['social_media_links'][0]['link']['href'], $actual->attr('href'));
-    $this->assertEquals(' ' . $test_data['social_media_links'][0]['link']['label'], $actual->text());
+    $section = $crawler->filter('footer.ecl-footer-core section.ecl-footer-core__section4');
+    $this->assertLinkList($section, 'core', $test_data['legal_navigation']);
+  }
 
-    $actual = $second_column->filter('ul li:nth-child(2) > a');
-    $this->assertEquals($test_data['social_media_links'][1]['link']['href'], $actual->attr('href'));
-    $this->assertEquals($test_data['social_media_links'][1]['link']['label'] . ' ', $actual->text());
+  /**
+   * Test European Commission footer standardised block rendering.
+   */
+  public function testEcFooterStandardisedBlockRendering(): void {
+    $this->configFactory->getEditable('oe_theme.settings')->set('branding', 'standardised')->save();
+    $test_data = [];
+    $html = $this->renderCorporateBlocksFooter('ec', $test_data);
```

Comment:
This method render the block and fetches the test data, isn't it more clear for one method to do one thing only? Let's simply:

```php
    $test_data = $this->getFixtureContent("{$type}_footer.yml");
    $html = $this->renderCorporateBlocksFooter($type, $test_data);
```